sdk/openai/inference/src/options/openai_client_options.rs

```diff
@@ -0,0 +1,45 @@
+use azure_core::ClientOptions;
+
+/// Options to be passed to [`OpenAIClient`](crate::clients::OpenAIClient).
+///
+/// Note: There are currently no options to be set.
+/// This struct is a placeholder for future options.
+// TODO: I was not  able to  find ClientOptions as a derive macros
+#[derive(Clone, Debug, Default)]
+pub struct OpenAIClientOptions {
+    pub(crate) client_options: ClientOptions,
+}
+
+impl OpenAIClientOptions {
+    /// Creates a new [`builders::OpenAIClientOptionsBuilder`].
+    pub fn builder() -> builders::OpenAIClientOptionsBuilder {
+        builders::OpenAIClientOptionsBuilder::new()
+    }
+}
+
+/// Builder to construct a [`OpenAIClientOptions`].
+pub mod builders {
+    use super::*;
+
+    #[derive(Clone, Debug, Default)]
+    pub struct OpenAIClientOptionsBuilder {
```

Comment:
You should also implement `azure_core::ClientOptionsBuilder`. See guidelines for details. We'll have a derive macro eventually to make this easier but aren't there yet.

Reply:
Same is true for client method options.